target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -154,6 +154,12 @@ ipq40xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0u@eth0" "2:lan:4" "3:lan:3" "4:lan:2" "0u@eth1" "5:wan"
 		;;
+		
+	zte,mf289f)
+		ucidef_set_interfaces_lan_wan "eth0" "eth1"
+		ucidef_add_switch "switch0" \
+			"0u@eth0" "2:lan:4" "3:lan:3" "4:lan:2" "0u@eth1" "5:wan"
```

Comment:
On the photos I see only two Ethernet ports, please double-check the switch configuration.

Reply:
Yes I confirm, I'll take a look in the next few days to fix it just as two ports